code42v2_connector.py

```diff
@@ -84,6 +90,21 @@ def __init__(self):
         self._username = None
         self._password = None
         self._client = None
+        self._proxy = None
+
+    def _is_ipv6(self, input_ip_address):
```

Comment:
this function name doesn't match the description / actual function logic. should it be something like `_is_valid_ip`?

Reply:
Fixed